app/main.py

```diff
@@ -1,2 +1,111 @@
+from typing import Any
+
+
+class HashNode:
+    def __init__(
+            self, key: int | float | complex | str | tuple | object,
+            value: Any
+    ) -> None:
+        self.key = key
+        self.value = value
+        self.hash = hash(self.key)
+
+
 class Dictionary:
-    pass
+
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.loaded = int(self.capacity * (2 / 3))
+        self.hash_table = [None] * self.capacity
+
+    def get_hash_table_load_info(self) -> str:
+        current_load = len(self.hash_table) - self.hash_table.count(None)
+        expected_load = self.loaded + 1
+        return (
+            "is Load" if current_load == expected_load else "is not Load"
+        )
```

Comment:
The logic in `get_hash_table_load_info` is incorrect. Instead of comparing `current_load` to `expected_load`, you should compare the current load factor (i.e., the ratio of non-None entries to the capacity) against a threshold (e.g., 2/3) to determine if the table is loaded.